hack/feature-wait.sh

```diff
@@ -13,7 +13,7 @@ if [ "$FEATURES" == "" ]; then
 fi
 
 all_ready=false
-export TEST_SUITES="validationsuite"
+export TEST_SUITES="validationsuite configsuite"
```

Comment:
typo in the commit: configusuit

Reply:
How comes we need this? Isn't the config step called after the validation?
The configuration step is meant to happen right after the validation and before running the suite